source/features/quick-repo-deletion.tsx

```diff
@@ -21,13 +21,19 @@ function handleToggle(event: delegate.Event<Event, HTMLDetailsElement>): void {
 		'.rgh-open-prs-of-forks' // PRs opened in the source repo
 	]);
 
-	if (hasContent && !confirm('This fork has open issues/PRs, are you sure you want to delete everything?')) {
+	if (hasContent && !confirm('This repo has open issues/PRs, are you sure you want to delete everything?')) {
 		// Close the <details> element again
 		event.delegateTarget.open = false;
-	} else {
-		// Without the timeout, the same toggle event will also trigger the AbortController
-		setTimeout(start, 1, event.delegateTarget);
+		return;
+	}
+
+	if (!pageDetect.isForkedRepo() && !confirm('⚠️ This action cannot be undone. This will permanently delete the repository, wiki, issues, comments, packages, secrets, workflow runs, and remove all collaborator associations.')) {
```

Comment:
Are you interested in adding a note that one can actually restore the repository (which can't be a fork) if it's deleted by mistake? https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/restoring-a-deleted-repository

Reply:
That’s pretty cool, we can add a “You can restore it” link on the confirmation text.